templates/external-secrets.yaml

```diff
@@ -115,6 +115,12 @@ spec:
                 {{`{{ .registry2Host }}`}}:
                   username: {{`{{ .registry2User }}`}}
                   password: {{`{{ .registry2Password }}`}}
+            tag-source:
+            - implementer: Git
+              config:
+                git-username: {{`{{ .gitUsername }}`}}
+                git-user-name: {{`{{ .gitUsername }}`}}
```

Comment:
Do we actually still need both of these?

Reply:
@jthompkins we can likely remove line 122. what do you think?

Reply:
testing removal now

Reply:
yeah it still works without user-name